File: Tests/NyxCoreTests/Fixtures/export/03-github-api.js
fetch("https://api.github.com/repos/nyx/nyx/issues?state=open&per_page=100", {
  method: "GET",
  headers: {
    "Authorization": `Bearer ${process.env.GITHUB_TOKEN}`,
    "Accept": "application/vnd.github+json",
    "X-GitHub-Api-Version": "2022-11-28",
  },
  redirect: "follow",
})
  .then((response) => response.json())
  .then((data) => console.log(data))
  .catch((error) => console.error(error));

File: Tests/NyxCoreTests/Fixtures/export/03-github-api.py
import os
import requests

response = requests.request(
    "GET",
    "https://api.github.com/repos/nyx/nyx/issues",
    params={
        "state": "open",
        "per_page": "100",
    },
    headers={
        "Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}",
        "Accept": "application/vnd.github+json",
        "X-GitHub-Api-Version": "2022-11-28",
    },
    allow_redirects=True,
)

print(response.status_code, response.text)
